plugins/cryptolib/cryptolib.cpp

```diff
@@ -570,7 +620,7 @@ CRYPTOLIB_API void CRYPTOLIB_CALL clPKIEncrypt(size32_t & __lenResult,void * & _
                                             size32_t lenInputdata,const void * inputdata)
 {
     verifyPKIAlgorithm(pkalgorithm);
-    Owned<CLoadedKey> publicKey = loadPublicKeyFromFile(publickeyfile, passphrase);
+    Owned<CLoadedKey> publicKey = g_KeyCache.getInstance(true, publickeyfile, nullptr, passphrase);
```

Comment:
Are there any threading issues with this keys being shared between multiple callers?  I would expect it to work, but double checking...